nengo_gui/static/netgraph.js

```diff
@@ -335,7 +335,7 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var uid = data.uid;
         for (var i = 0; i < Nengo.Component.components.length; i++) {
             if (Nengo.Component.components[i].uid === uid) {
-                Nengo.Component.components[i].remove(true);
+                Nengo.Component.components[i].remove(true, data.report_back);
```

Comment:
Hmm..  the intent is that it's a flag to indicate whether the NetGraph should report back to the server that the removal happened.  We don't want it to do so in this case, as the server is the one who asked for it to be removed in the first place.  Not sure what a better name would be.